Examples/UICatalog/Scenarios/SendKeys.cs

```diff
@@ -39,7 +39,7 @@ public override void Main ()
 
         txtResult.KeyDown += (s, e) =>
                              {
-                                 rKeys += (char)e.KeyCode;
+                                 rKeys += new Rune ((uint)(e.KeyCode & ~KeyCode.AltMask & ~KeyCode.CtrlMask & ~KeyCode.ShiftMask));
```

Comment:
Please use `Key`, not `KeyCode`. `Key` has helpers for this. Since this is an example, we don't want to be encouraging devs to use `KeyCode`.

Reply:
Done in [14c68e5](https://github.com/gui-cs/Terminal.Gui/pull/4225/commits/14c68e550f04bce301b1e297d93e65ea1fc76070)